packages/cozy-harvest-lib/src/services/biWebView.spec.js

```diff
@@ -69,7 +69,7 @@ describe('handleOAuthAccount', () => {
       t,
       account: {
         ...account,
-        ...{ auth: { bankId: TEST_BANK_COZY_ID } },
+        ...{ auth: { bankIds: [TEST_BANK_COZY_ID] } },
```

Comment:
Are we updating the io.cozy.accounts to add this `bankIds` fields? We should definitely updates this documentaiton https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.accounts.md#caisse-d%C3%A9pargne-linxo-connector

Reply:
The auth attribute is more like a "free" field to have any authentication fields specific to the associated connector. But yes, this document needs an update